### PR TITLE
PhoneField styled-component migration

### PR DESCRIPTION
**Pull request by wakooka**
image

### CODE REVIEW

File: src/phoneField/PhoneField.tsx [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+  }
+}
+
+type errorField = string | JSX.Element
+interface PhoneFieldProps {
+  readonly name: string
+  readonly onChange: (obj: PhoneFieldOnChangeParameters) => void
+  readonly id?: string
+  readonly className?: Classcat.Class
+  readonly extraClassName?: Classcat.Class
```
Comment (grosbouddha) [inline src/phoneField/PhoneField.tsx]: extraClassName makes me think that is will be on the same element as className (i.e. the root).
Can we use a naming that tries to identify where the class is added ?
In that case, maybe innerWrapperClassName ?

File: src/autoComplete/AutoCompleteList.unit.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import { shallow, mount } from 'enzyme'
 
-import AutoCompleteList from './AutoCompleteList'
+import AutoCompleteList from './autoCompleteList'
```
Comment (pith) [inline src/autoComplete/AutoCompleteList.unit.tsx]: It brokes the test
```
Test suite failed to run
    Cannot find module './autoCompleteList' from 'AutoCompleteList.unit.tsx'
      1 | import React from 'react'
      2 | import { shallow, mount } from 'enzyme'
    > 3 | 
        | ^
      4 | import AutoCompleteList from './autoCompleteList'
      5 | import ItemChoice, { ItemChoiceStatus } from 'itemChoice'
      6 | 
```

Comment (wakooka) [reply src/autoComplete/AutoCompleteList.unit.tsx]: Oops bad conflict resolving.

File: src/phoneField/PhoneField.tsx (hunk 1 above)
Comment (wakooka) [reply src/phoneField/PhoneField.tsx]: Done PTAL

File: src/autoComplete/AutoCompleteList.unit.tsx (hunk 2 above)
Comment (wakooka) [reply src/autoComplete/AutoCompleteList.unit.tsx]: Fixed

**Pull request merged**